src/style.css

```diff
@@ -0,0 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search-bar {
+  width: 100%;
+}
+
+::placeholder {
+  color: #3d4e61;
+}
+
+.big {
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19;
+  
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+
+  font-weight: 300px;
+  font-size: 16px;
```

Comment:
There are a lot of similar styles in .big and .small. Move them under one class.
Also, give more clear class names, such as big-input, small-input.